osc-ui/src/main/java/org/osc/core/broker/view/maintenance/PluginsLayout.java

```diff
@@ -161,6 +160,17 @@ private Button getDownloadSdkButton() throws URISyntaxException, MalformedURLExc
         return downloadSdk;
     }
 
+    private Button getDownloadSdkButtonforManager() throws URISyntaxException, MalformedURLException {
```

Comment:
For consistency can you rename the method above getDownloadSdkButtonForSdnController?